app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -45,7 +45,7 @@ def instances
       # saving hardware information (CPU, Memory, etc.)
       ps_hw = persister.hardwares.build(
         :vm_or_template  => ps_vmi,
-        :cpu_total_cores => Float(instance["processors"]).ceil,
+        :cpu_total_cores => Float(instance['virtualCores']['assigned']),
```

Comment:
`instance['virtualCores']['assigned']` should already be an integer

https://cloud.ibm.com/apidocs/power-cloud#pcloud-pvminstances-get

Reply:
the column is original a Float Jay. So anyways it will be typecast automatically to Float. Unless we want to change that.